<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeclusteringDialog</class>
 <widget class="QDialog" name="DeclusteringDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Declustering tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Declustering computes a new variable that is called the declustering weight. Declustering weights are used to get location unbiased statistics like mean and histogram, which can be used on geostatistics workflows so the results reflect the actual statistics. This is specially true for data sets that are not regularly spaced.&lt;/p&gt;&lt;p&gt;If the sampling prefered high values, for example gold grade, then one must look for the lowest mean while running declus. If lower values (e.g. water saturation in petroleum projects) were the target of the sampling, then one must find the weights that yield the highest mean.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Declustering xxxxxx</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="margin">
      <number>30</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>1) Configure and run declustering:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnParameters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>edit parameters</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/setting16</normaloff>:/icons/setting16</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>2) Evaluate the result:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSummary">
         <property name="toolTip">
          <string>view summary</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/summary16</normaloff>:/icons/summary16</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnHistogram">
         <property name="toolTip">
          <string>open resulting histogram</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/histo16</normaloff>:/icons/histo16</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLocmap">
         <property name="toolTip">
          <string>location map of resulting weights</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/locmap16</normaloff>:/icons/locmap16</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>3) If good, save the computed weights:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="toolTip">
          <string>save data with declustering weights</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/save16</normaloff>:/icons/save16</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>DeclusteringDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnParameters</sender>
   <signal>clicked()</signal>
   <receiver>DeclusteringDialog</receiver>
   <slot>onDeclus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSummary</sender>
   <signal>clicked()</signal>
   <receiver>DeclusteringDialog</receiver>
   <slot>onViewSummary()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHistogram</sender>
   <signal>clicked()</signal>
   <receiver>DeclusteringDialog</receiver>
   <slot>onHistogram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>DeclusteringDialog</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLocmap</sender>
   <signal>clicked()</signal>
   <receiver>DeclusteringDialog</receiver>
   <slot>onLocmap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDeclus()</slot>
  <slot>onViewSummary()</slot>
  <slot>onHistogram()</slot>
  <slot>onSave()</slot>
  <slot>onLocmap()</slot>
 </slots>
</ui>
